.travis.yml

```diff
@@ -24,10 +24,12 @@ matrix:
           env: TOXENV=pypy
         - python: pypy3
           env: TOXENV=pypy3
-        - python: 3.5
-          env: TOXENV=py2-cover,py3-cover,coverage
+        - python: 3.6
+          env: TOXENV=py36-cover,coverage
```

Comment:
Unless you are also dropping Python 2 in this commit, testing needs to be run against Python 2/3 to get full coverage results.